src/Options.php

```diff
@@ -616,7 +616,7 @@ private function configureOptions(OptionsResolver $resolver): void
             'sample_rate' => 1,
             'mb_detect_order' => null,
             'attach_stacktrace' => false,
-            'context_lines' => 3,
+            'context_lines' => 5,
```

Comment:
@HazAT which should be the right default for this value? Which is the "normal" default on the server side?